src/sagas/mixins.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {
+  takeEvery,
+  select,
+  all,
+  put,
+  call,
+} from 'redux-saga/effects';
+import { t } from 'ttag';
+import {
+  types,
+  selectAddressAddressesSuccess,
+  selectAddressAddressesFailure,
+  onExceptionCaptured,
+} from '../actions';
+import { logger } from '../logger';
+import { getAllAddresses } from '../utils';
+
+const log = logger('mixins');
+
+export function* fetchAllWalletAddresses() {
```

Comment:
Why is this a "mixin"?

Reply:
Because there is a pattern for the sagas, they are being used in cycles and major features that also have cycles. This one doesn't belong to a cycle nor to a major feature. Therefore I created this space for standalone effects. I should aggregate future standalone effects as well.